docs/concept/buckconfig.soy

```diff
@@ -2073,6 +2073,16 @@ cxx_library(
   {/param}
 {/call}
 
+{call buckconfig.entry}
+  {param section: 'intellij' /}
+  {param name: 'extra_compiler_output_modules_path' /}
+  {param example_value: 'buck-out/extra-intellij-output' /}
+  {param description}
+    This flag decides whether or not extra library modules will be generated for Android library
```

Comment:
This is not a flag, right? Also, it seems generic enough, not Android specific.

How about something like this:

> This option specifies the location of additional modules for code generated outside of buck graph. For example, it can be used to specify the location of R.java classes generated for Android plugin to help Layout Preview with resolving references to resources.